src/vos/lru_array.c

```diff
@@ -271,6 +279,12 @@ lrua_array_alloc(struct lru_array **arrayp, uint32_t nr_ent, uint32_t nr_arrays,
 	if (array == NULL)
 		return -DER_NOMEM;
 
+	if (flags & LRU_FLAG_TRACK_ALLOC) {
+		tls = vos_tls_get();
+		d_tm_inc_counter(tls->vtl_lru_alloc_size,
```

Comment:
minor nit, we could potentially pass in the metric pointer rather than using a flag.  This would allow us to track more than one usage.  Currently, we use it for timestamp cache and DTX local id assignment.

Reply:
sure thanks